modules/navigation.js

```diff
@@ -0,0 +1,27 @@
+const navList = document.querySelector('.nav-list');
+const navAdd = document.querySelector('.nav-add');
+const navContact = document.querySelector('.nav-contact');
+const listSection = document.querySelector('.list');
+const addANewSection = document.querySelector('.add-a-new-book');
+const contactSection = document.querySelector('.contact');
+
+navList.addEventListener('click', () => {
+  listSection.classList.remove('hide');
+  addANewSection.classList.add('hide');
+  contactSection.classList.add('hide');
+});
+
+navAdd.addEventListener('click', () => {
+  listSection.classList.add('hide');
+  addANewSection.classList.remove('hide');
+  contactSection.classList.add('hide');
+});
+
+navContact.addEventListener('click', () => {
+  listSection.classList.add('hide');
+  addANewSection.classList.add('hide');
+  contactSection.classList.remove('hide');
+});
```

Comment:
- [ ] the navigation part is not working entirely, when clicking on any of the links the screen doesn't change, that is due to you not changing the `display` actively to block instead of not, you can solve it by doing the following:
```
navList.addEventListener('click', () => {
  listSection.classList.add('unhide');
  addANewSection.classList.remove('unhide');
  addANewSection.classList.add('hide');
  contactSection.classList.remove('unhide');
  contactSection.classList.add('hide');
});

navAdd.addEventListener('click', () => {
  listSection.classList.remove('unhide');
  listSection.classList.add('hide');
  addANewSection.classList.add('unhide');
  contactSection.classList.remove('unhide');
  contactSection.classList.add('hide');
});

navContact.addEventListener('click', () => {
  listSection.classList.remove('unhide');
  listSection.classList.add('hide');
  addANewSection.classList.remove('unhide');
  addANewSection.classList.add('hide');
  contactSection.classList.add('unhide');
});
```
keep in mind that you also need to create an additional CSS class called unhide in this scenario and you should include the `display: block` attribute in that class.

Reply:
Thank you very much for the feedback! It's super weird because in my browser the links are fully functional, or what do you mean when clicking on any of the links the screen doesn't change? Would you be so kind to check it again and tell me if it was not a loading error? In chrome it runs super well :), anyway I will take your feedback into account and I thank you

Reply:
@mikethreels I wish you would allow yourself to follow up on your revisions, unfortunately trying to respond to your suggestions I lost several hours of valuable time that I could have invested in other activities, in the end, I re-uploaded the work exactly the same way it was when you posted it you reviewed and it was approved because it works as it should, the feedback is as valuable from one side as from the other.